docs/content/advanced/backend/migrations.md

```diff
@@ -0,0 +1,85 @@
+# Migrations
+
+In this document, you’ll learn about what Migrations are, their purpose, how you can run them, and how you can create your own Migrations.
+
+:::note
+
+Medusa’s Migrations do not work with SQLite databases. They are intended to be used with PostgreSQL databases, which is the recommended Database for your Medusa production server.
+
+:::
+
+## Overview
+
+Migrations in Medusa are scripts that are used to make additions or changes to your database schema. They are essential for both when you first install your server and for subsequent server upgrades later on.
+
+When you first install your Medusa server, the database schema used must have all the tables necessary for the server to run. This is automatically done when you run the `seed` command which also seeds your database with dummy data:
```

Comment:
comment: Would be great to add both approaches of running migrations.

```suggestion
When you first create your Medusa server, the database schema must have all the tables necessary for the server to run. This can be done in one of two ways: 
- medusa migrations run -> add a couple of sentences...
- medusa seed -> migrations + dummy data...
```